app/controllers/admin/statistics_announcement_publications_controller.rb

```diff
@@ -0,0 +1,58 @@
+class Admin::StatisticsAnnouncementPublicationsController < Admin::BaseController
+  before_action :find_statistics_announcement
+  layout "design_system"
+
+  def index
+    if params[:title].present?
+      filter
+    end
+  end
+
+  def connect
+    @statistics_announcement.assign_attributes(publication_params)
+
+    if @statistics_announcement.save
+      redirect_to [:admin, @statistics_announcement], notice: "Announcement updated successfully"
+    else
+      filter
+      render :index
+    end
+  end
+
+private
+
+  def filter
+    @filter ||= Admin::EditionFilter.new(Edition, current_user, edition_filter_options)
+  end
+
+  def params_filters
+    params.slice(:title, :page)
+          .permit!
+          .to_h
+  end
+
+  def params_filters_with_default_state
+    params_filters.reverse_merge("state" => "active")
+  end
+
+  def edition_filter_options
+    params_filters_with_default_state
+                       .symbolize_keys
+                       .merge(
+                         include_unpublishing: true,
+                         include_link_check_reports: true,
+                         include_last_author: true,
```

Comment:
sorry i should've asked this before, but are we using these 3 params

```
include_unpublishing: true,
include_link_check_reports: true,
include_last_author: true,
```

Reply:
Yes, double checked this agains the documents search controller, and it doesn't look like they are necessary, so I have removed them.